src/routes/Apps/App/TrainDialogs.tsx

```diff
@@ -157,6 +157,8 @@ interface TranscriptImportData {
     warnings: string[]
     // Conditions are keyed by TrainScorerStep.importId.
     conditions?: { [key: string]: CLM.Condition[] }
+    // Expected entities are keyed by actionId.
+    expectedEntityNames?: { [key: string]: string }
```

Comment:
I would prefer names like `actionIdToEntityNameMap`.  Then you don't need the comment as it's self explanatory.  As it is the name is misleading.

I used names that are plural to mean it's a list. 
```
entityName: 'myEntity',
entityNames: ['entityOne', 'entityTwo']
```

This name makes it sound like it should be a list of strings, but it's not.

Reply:
Done